docs/topics/ref_proficiency-levels.adoc

```diff
@@ -2,7 +2,7 @@
 [id='proficiency-levels']
 = Proficiency levels
 
-Each mission available on {launcher} teaches you about certain topics, but requires certain minimum knowledge, which varies by mission. For clarity, the minimum requirements and concepts are organized in several proficiency levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
+Each mission available on {launcher} teaches you about certain topics, but requires certain minimum knowledge, which varies by mission. The minimum requirements and concepts are organized in several proficiency levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
```

Comment:
Each available mission teaches certain concepts that require a minimum of background knowledge (this is really vague, how do you know what you don't know?). The minimum requirements and concepts are organized into different levels of proficiency. In addition tot he concepts described here, you may need additional information specific to each mission. (not sure this is much smoother, but I tried)

Reply:
Will fix with some modifications.